public/locales/fi/signup.json

```diff
@@ -99,6 +99,7 @@
     "labelFirstName": "Etunimi",
     "labelLastName": "Sukunimi",
     "labelPhoneNumber": "Puhelinnumero",
+    "labelPriceGroup": "Hintaryhmä",
```

Comment:
The Figma prototype uses the term "Asiakasryhmä" - was that maybe agreed to be changed to "Hintaryhmä" or is this something to change?

Reply:
Changed "Hintaryhmä" to "Asiakasryhmä"